addons/mass_mailing/tests/test_mailing_internals.py

```diff
@@ -124,21 +124,21 @@ def patched_images_to_urls(self, b64images):
                                 <div style="color: red; background-image: url(data:image/jpg;base64,{BASE_64_STRING}16); background: url('data:image/jpg;base64,{BASE_64_STRING}17'); display: block;"/>
                             <![endif]-->
                             <img src="data:image/png;base64,{BASE_64_STRING}0">
-                        </body></html>
+                        </section>
                     """,
                 })
         self.assertEqual(len(attachments), 19)
         self.assertEqual(attachments[0]['id'], attachments[18]['id'])
-        self.assertEqual(str(mailing.body_html), f"""
-                        <html><body>
-                            <img src="/web/image/{attachments[0]['id']}?access_token={attachments[0]['token']}">
-                            <img src="/web/image/{attachments[1]['id']}?access_token={attachments[1]['token']}">
-                            <div style='color: red; background-image:url("/web/image/{attachments[2]['id']}?access_token={attachments[2]['token']}"); display: block;'></div>
-                            <div style="color: red; background-image:url('/web/image/{attachments[3]['id']}?access_token={attachments[3]['token']}'); display: block;"></div>
-                            <div style='color: red; background-image:url("/web/image/{attachments[4]['id']}?access_token={attachments[4]['token']}"); display: block;'></div>
-                            <div style='color: red; background-image:url("/web/image/{attachments[5]['id']}?access_token={attachments[5]['token']}"); display: block;'></div>
-                            <div style="color: red; background-image:url(/web/image/{attachments[6]['id']}?access_token={attachments[6]['token']}); display: block;"></div>
-                            <div style="color: red; background-image: url(/web/image/{attachments[7]['id']}?access_token={attachments[7]['token']}); background: url('/web/image/{attachments[8]['id']}?access_token={attachments[8]['token']}'); display: block;"></div>
+        self.assertEqual(str(mailing.body_html).strip(), f"""
+                        <section>
+                            <img src="/web/image/{attachments[0]['id']}?access_token={attachments[0]['token']}"/>
+                            <img src="/web/image/{attachments[1]['id']}?access_token={attachments[1]['token']}"/>
+                            <div style="color: red; background-image:url(&quot;/web/image/{attachments[2]['id']}?access_token={attachments[2]['token']}&quot;); display: block;"/>
```

Comment:
Did we actually need to change these for some reason, or just cleanup?

Reply:
It is the sanitizer who sanitizes stuff, changes quoting, ... those tests are quite low level so unsure this really changes anything in real life.

Reply:
Ah right this is the output :man_facepalming: 

Fine then, yes it should be perfectly equivalent assuming no bug in lxml..